### ISSUE TITLE
Add ability to check if a NodeElement is disabled

### ISSUE THREAD

**Issue by evangelion1204** [minkphp/Mink#521]
Hi,

I would like to implement a method called `isDisabled` for the NodeElement to check if an element is disabled or not. This can come quite handy because in AjaxRequest you often disable Buttons/Elements.
This could be done quite analog to `isChecked`. Is this new function wished and how could it be named? `isDisabled` or `isEnabled` - cause you normally add the attribute `disabled` to an element `isDisabled` seems more logical, from the properties `isEnabled`.

Thanks,

Michael

**Comment by TerjeBr** [comment]
Is there any progress on adding the "isDisabled" method. (I was needing it today.)

**Comment by aik099** [comment]
None. If any progress would happen, then it will be on #499.

**Comment by TerjeBr** [comment]
Why does this depend on #499 ?
As you have driver dependent implementations of "isChecked" you should probably have driver dependent implementations of "isDisabled".

I see #499 as just an additional "nice to have".

**Comment by TerjeBr** [comment]
@stof said in https://github.com/minkphp/Mink/issues/521#issuecomment-42497277 "I would not add getNodeProperty". I understood that to mean that "isDisabled" still could go ahead.

**Comment by aik099** [comment]
@TerjeBr , because in this issue discussion we've agreed that instead of just making `getNodeProperty` method we'd better implement a way to execute a js on a given element for all js-enabled drivers. That's what #499 is all about by the way.

**Comment by aik099** [comment]
> https://github.com/minkphp/Mink/issues/521#issuecomment-330498385

@TerjeBr , the discussion happened 3 years ago. I've honestly don't remember what was it. This could also mean, that `NodeElement::isDisabled` method could be implemented using functionality described in #499 instead of adding new `getNodeProperty` method. Yet we need to have a fallback code for non-js enabled drivers too.

In any case you've welcome to send a PR.

**Comment by vijaycs85** [comment]
I might be wrong, by the look of it, we already have `executeScript()` and `evaluateScript()` in Behat\Mink\Driver\DriverInterface. All we need is an implementation of executeScript() (and fallback as mentioned by @aik099) and `NodeElement::isDisabled` method to use  `executeScript()` to get disabled status. This way it wouldn't be [just DOM](https://github.com/minkphp/Mink/issues/521#issuecomment-42497277).
